src/radix_connect/mobile/relay_service/success_response.rs

```diff
@@ -0,0 +1,65 @@
+use crate::prelude::*;
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct SuccessResponse {
+    pub method: String,
+    /// The unique id of the session established with the dApp.
+    pub session_id: SessionID,
+    /// Wallet's public key to be used to create the shared secret with the dApp.
+    pub public_key: KeyAgreementPublicKey,
+    /// Hex encoded WalletInteractionResponse
+    pub data: String,
```

Comment:
What is `data`? doc says: "`Hex encoded WalletInteractionResponse"`, so this should be `BagOfBytes` right?